sound/soc/sof/intel/cnl.c

```diff
@@ -246,5 +246,7 @@ const struct sof_intel_dsp_desc cnl_chip_info = {
 	.ipc_ack_mask = CNL_DSP_REG_HIPCIDA_DONE,
 	.ipc_ctl = CNL_DSP_REG_HIPCCTL,
 	.rom_init_timeout	= 300,
+	.ssp_count = 8,
+	.ssp_base_offset = CNL_SSP_BASE_OFFSET,
```

Comment:
separate patch, this has nothing to do with slave mode.

Reply:
OK, thanks!